resolwe/storage/connectors/transfer.py

```diff
@@ -284,6 +284,9 @@ def transfer(
             stream = to_connector.open_stream(to_base_url / to_url, "wb")
             from_connector.get(from_url, stream, chunk_size=chunk_size)
             stream.close()
+            # Store computed hashes as metadata for later use.
+            to_connector.set_hashes(to_base_url / to_url, hashes)
```

Comment:
Merge this commit with the previous one, so it will be obvious that you've moved the same chunk of code.